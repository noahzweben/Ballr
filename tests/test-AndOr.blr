gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

	print (true && false);
    print (true && true); 
    print (false && false); 
    print (true || false); 
    print (true || true); 
    print (false || false); 

  	}
}