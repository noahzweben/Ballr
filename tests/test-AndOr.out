40
20
10
SDL could not initialize! SDL Error: No available video device
Window initialization failed.